xero-finance.yaml

```diff
@@ -1028,6 +1028,84 @@ paths:
                 title: InvalidRequest
                 status: 400
                 detail: Organisation xxx does not exist
+  /bankstatementsplus/statements:
+    get:
+      security:
+        - OAuth2: [finance.bankstatementsplus.read]
+      tags:
+        - BankStatementsPlus
```

Comment:
Can you change the tag from BankStatementsPlus to Finance so it matches the tag the rest of the finance endpoints have?